docs/FAQ.md

```diff
@@ -18,12 +49,59 @@ Only Delphi source files under the base directory (including in subdirectories)
 
 ### "Analyze All Open Files" does not analyze my `.dpr` or `.dpk` file, even though it is open.
 
-This is intentional, as analyzing `.dpr` and `.dpk` files typically raises a large number of erroneous issues due to
+This is intentional, as analyzing `.dpr` and `.dpk` files may raise erroneous issues due to
 dependency analysis limitations. `.dpr` and `.dpk` files can be explicitly analyzed using "Analyze This File".
 
 ### DelphiLint has been stuck in analysis for a long time.
 
 Generally speaking, DelphiLint analyses can take upwards of 30 seconds when dealing with files with many imports. If it
 has been a longer time, check the progress of the scan in the logs at
 `%APPDATA%\DelphiLint\logs\delphilint-server.log`. If a problem seems to have occurred, the server can be restarted
-with `DelphiLint > Restart Server`.
\ No newline at end of file
+with `DelphiLint > Restart Server`.
+
+### When I go to analyze a file, a dialog appears saying "Could not connect to the configured SonarQube instance."

```

Comment:
"When I go to analyze a file, a dialog appears saying" can be changed to "Error:"

Because this line is currently so long, it ends up wrapping and looking a bit bad in the rendered markdown.